### DISCUSSION TITLE
datadog default_zero() function isnt working in terragrunt

### DISCUSSION THREAD

**Discussion by Junaidfarook** [gruntwork-io/knowledge-base#840] [How do I]
monitors = {
    "CF-origin-latency" = {
      name = "cloudfront origin latency"
      query = {
        
        type = "metric_standard"
         
        params = {
         
          time_aggr   = "avg"
          time_window = "current_1mo" 
          metric   = "aws.cloudfront.origin_latency"
          tags     = "aws_account:1231,environment:staging,application:abc"
          operator = ">"
          key      = "name"
          function = "default_zero(aws.cloudfront.origin_latency{*})"
        }
      }
      
      #### upon running it says 400 Bad Request: {"errors":["The value provided for parameter 'query' is invalid"]}
     
      any idea??

**Comment by denis256** [comment]
Hello,
not 100% sure why it fails, but I prepared a small example of how to use `default_zero` as string:
https://github.com/denis256/terragrunt-tests/tree/master/discussion-840

To do more troubleshooting can I think can be used used approach from debugging page https://terragrunt.gruntwork.io/docs/features/debugging/
